tcms/report/views.py

```diff
@@ -572,6 +584,19 @@ def get(self, request, *args, **kwargs):
         return view(request, *args, **kwargs)
 
 
+def _get_testers_assignees_ids(case_runs):
+    testers_ids = set()
+    assignees_ids = set()
+    for case_run in case_runs:
+        primary_key = case_run.tested_by_id
+        if primary_key:
```

Comment:
I think you can inline the `primary_key` variable here, as it is used only in one place and does not contain any real information about the code and its purpose. Same goes for the one below.